Current_DAC/AidType.xml

```diff
@@ -71,6 +71,45 @@
             <category>B</category>
             <dac:reference>DCD/DAC/STAT(2009)1;DCD/DAC/STAT(2018)42</dac:reference>
         </codelist-item>
+        <codelist-item status="active" activation-date="2021-01-01">
+            <code>B031</code>
```

Comment:
@nosvalds this looks correct,. What I am realising now is that the new codes are sub-categories of B03, which we have not had before: https://iatistandard.org/en/iati-standard/203/codelists/aidtype/

How would the current PR look on this page?: https://iatistandard.org/en/iati-standard/203/codelists/aidtype/

As we replicate codes exactly as they are for the OECD DAC my thinking is to keep them on the [Aid Type page](https://iatistandard.org/en/iati-standard/203/codelists/aidtype/
) and list after B03, rather than created another page for sub-category? @nosvalds  Is that how the current PR is set?
@akmiller01 @IATI/bas unless there are any other ideas/suggestions?

Reply:
@PetyaKangalova I believe Alex M. answered your question here on the Standup yesterday. The code will show up on the same page. 

Once we merge this to `master` I will regenerate the website pages and push them to the `dev` environment for UAT review before pushing the pages into the live website.